src/screens/AddTrip/AddTrip.tsx

```diff
@@ -0,0 +1,236 @@
+                                leftIcon={
+                                    <Icon
+                                        style={styles.icon}
+                                        name="comment"
+                                        size={24}
+                                        color="black"
+                                        type="font-awesome-5"
+                                    />
+                                }
+                                leftIconContainerStyle={styles.leftIcon}
+                                value={values.description}
+                                onChangeText={handleChange("description")}
+                                onBlur={handleBlur("description")}
+                                multiline={true}
+                                numberOfLines={4}
+                                style={styles.textArea}
+                            />
+                            {error && (
+                                <Text style={styles.errorText}>
+                                    {error.message}
+                                </Text>
+                            )}
+                            <Button
+                                containerStyle={styles.buttonContainer}
+                                buttonStyle={styles.submitButton}
+                                title={t("screens.add_trip.submit_add_trip")}
+                                titleStyle={{
+                                    color: "black",
+                                    fontSize: 25,
+                                }}
+                                icon={
+                                    <Icon
+                                        style={styles.iconButton}
+                                        name="arrow-right"
+                                        size={15}
+                                        color="black"
+                                        type="font-awesome-5"
+                                    />
+                                }
+                                iconRight={true}
+                                onPress={() => handleSubmit()}
+                                loading={loading}
+                            />
+                        </>
+                    )}
+                </Formik>
+                <Button
+                    containerStyle={styles.buttonContainer}
+                    type="clear"
+                    title={t("screens.add_trip.stay_home")}
+                    titleStyle={{ color: "darkslategray" }}
+                    onPress={() => navigation.navigate("Dashboard")}
+                />
+            </SafeAreaView>
+        </ScrollView>
+    );
+};
+// eslint-disable-next-line
+const validationSchema = (t: TFunction): object => {
+    return Yup.object().shape({
+        tripName: Yup.string()
+            .min(1)
+            .required(t("validation.tripNameRequired")),
+        description: Yup.string()
+            .min(1)
```

Comment:
maybe also a limit here would make sense I guess

Reply:
What do you think of 200characters for the trip description (copied from prototype)?

Reply:
Yes i think this should work